dbf2csv/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+"access_conversions_user","Manage dbf to csv conversions","model_dbftocsv_conversion","base.group_user",0,0,0,0
+"access_conversions_manager","Manage dbf to csv conversions","model_dbftocsv_conversion","base.group_system",1,1,1,1
```

Comment:
Do you really need to be a sysadmin to  perform a file conversion?

Reply:
IMHO yes.
This module helps to convert SAGE .dbf files to .csv in order to import them later.
DBF files contain confidential information about the company: customers, products, pricelists, invoices, ... Therefore, from the moment they are stored in a model, it is necessary to protect them.